packages/components/psammead-paragraph-enzyme/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,261 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Mounting, shallow and render tests Paragraph tested with Enzyme shallow render should render correctly with no props 1`] = `
+ShallowWrapper {
+  Symbol(enzyme.__root__): [Circular],
```

Comment:
These snapshots will be alot better looking if you add
```
"jest": {
  "snapshotSerializers": ["enzyme-to-json/serializer"]
}
``` from https://medium.com/codeclan/testing-react-with-jest-and-enzyme-20505fec4675